packages/lime-system/files/usr/lib/lua/lime/network.lua

```diff
@@ -68,7 +68,10 @@ function network.primary_address(offset)
 	invalid = invalid or ipv4_template:equal(mc:minhost())
 	--! Generated address is the broadcast address like 192.0.2.255/24 ?
 	invalid = invalid or ipv4_template:equal(mc:broadcast())
-	if invalid then ipv4_template = mc:maxhost() end
+	if invalid then
+		ipv4_template = mc:maxhost()
```

Comment:
Are you sure this will not cause easier collision in common situation?

Reply:
Do you mean that the lack of prefix resulting in a /32 was intentional?
I'm not sure I got your point, do you mean that there will be a lot of collisions on the maxhost() IP? In this case, sure there will be, but for avoiding this we should entirely redesign this section of the code. If you think my PR is deepening the problem, please explain this point.